deploy/crds/kieapp.crd.yaml

```diff
@@ -163,7 +163,7 @@ spec:
                         format: int32
                         type: integer
                       url:
-                        description: LDAP Endpoint to connect for authentication
+                        description: LDAP endpoint to connect for authentication. For failover set two or more LDAP endpoints separated by space.
```

Comment:
We now generate the crd automatically. Instead of changing the description here... change the comment directly above the ‘url’ field, in the api strict.

Reply:
Then run ‘make test’ and commit changes

Reply:
Oh yeah, I did forget about it, thanks.

Reply:
updated.